upgrade/UPGRADE-v11.0.0-dev.md

```diff
@@ -881,6 +881,11 @@ There you can find links to upgrade notes for other versions too.
     - see #project-base-diff to update your project
 - old phpRedisAdmin has been replaced with new Redis Commander ([#2550](https://github.com/shopsys/shopsys/pull/2550))
     - see #project-base-diff to update your project
+- product ordering in all product lists has been changed, now are products ordered primary by availability dispatch time. 
+    In stock as first and out of stock as last.  ([#2555](https://github.com/shopsys/shopsys/pull/2555))
+  - A prerequisite for modifying the sorting is canceling the nullable property for dispatch time in availability. 
+  - Now the maximum value of 65535 is assigned to out of stock availability.
+  - see #project-base-diff to update your project
```

Comment:
We are supposed to write UPGRADE notes as directives what to do, you are describing changes here and it should not be part of UPGRADE notes.